share/spice/islamic_prayer_times/islamic_prayer_times.js

```diff
@@ -0,0 +1,97 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_islamic_prayer_times = function(api_result) {
+
+        if (!api_result || api_result.error || api_result.status_code === 0 || !api_result.items) {
+            return Spice.failed('islamic_prayer_times');
+        }
+
+        DDG.require('moment.js', function() {
+
+            function setDst(time, dst) {
```

Comment:
Let's move these functions out of the callback. They can be moved above `env.ddg_spice_islamic_prayer_times`.

We call them from inside the closure, where `moment` is defined, so its OK :+1: